src/com/flashfla/net/Multiplayer.as

```diff
@@ -595,6 +619,12 @@ package com.flashfla.net
          */
         public function createRoom(name:String, password:String = "", maxUsers:int = 2, maxSpectators:int = 100):void
         {
+            if (currentUserRoomCount() > 1)
+            {
+                Alert.add("ERROR: Cannot join two rooms at a time.", 120);
+                return;
+            }
+            
             if (connected && name)
```

Comment:
Shouldn't this be a simple guard clause ?
`if (!connected || !name) return;`
Because `sendRoomVariables` should not be called if that guard returns.